website/docs/secrets/setup-secrets-operator.mdx

```diff
@@ -3,12 +3,125 @@ title: Setup Secrets Operator
+        ![ESO PR](./imgs/eso3.jpg)
+    - Once Flux reconcile the git source this should take few minutes, you will find the operator installed and the secrets lives in the cluster
+- To bootstrap the operator on a leaf Cluster
+    - Go to `Clusters` from the side bar and `Create A Cluster`
+    - Select your template (must have the enable profiles annotation)
+        ![ESO Cluster](./imgs/eso5.jpg)
+    - Select `External Secrets` from and input your `namespace`
+        ![ESO Profile](./imgs/eso4.jpg)
+    - Then Adjust the `values` according to your secrets git repository
+        ![ESO Values](./imgs/eso2.jpg)
+    - After that Authenticate with Github, Create and merge the pull request
+        ![ESO PR](./imgs/eso3.jpg)
+    - Once the cluster is created and Flux reconcile the git source this should take few minutes, you will find the operator installed and the secrets lives in the cluster
+
 
 ## Set up SOPS
-// Explain how to set up SOPS on leaf clusters. 
\ No newline at end of file
+
+- Installing Sops on Managmenet cluster, follow [this guide](/secrets/setup-sops.mdx)
+
+### Bootstrapping Sops to leaf clusters
+
+import SopsBootstrapJob from "!!raw-loader!./assets/sops-bootstrap-job.yaml";
+import TemplateParams from "!!raw-loader!./assets/template-params.yaml";
+import TemplateAnnotations from "!!raw-loader!./assets/template-annotations.yaml";
+
+- **This will need the following**
+    - `ClusterBootstrapConfig` job to install sops and generate keys
+        We will need a job to run in the bootstrapping of leaf cluster to install sops and generate the keys,the following example is using GPG encryption
+        
+        - The Job CR, follow [this guide](/cluster-management/getting-started/#add-a-cluster-bootstrap-config) for more information on ClusterBootstrapConfig
+
+            <details><summary>Expand to view </summary>
+                <CodeBlock
+                title="clusters/management/capi/boostrap/sops-bootstrap-job.yaml"
+                className="language-yaml"
+                >
+                    {SopsBootstrapJob}
+                </CodeBlock>
+            </details>
+
+        - Template Updates
+
+            In order to install sops we need some modifications to the template to allow it to create a [Kustomization](https://fluxcd.io/flux/guides/mozilla-sops/#configure-in-cluster-secrets-decryption) for reconciling the secrets on the cluster for sops during cluster creation for the secrets path and run the `ClusterBootstrapConfig` job
+
+            It will refer to `sops` location in the root of cluster files
+
+            - Template Updates
+                - It should have annotation `templates.weave.works/sops-enabled: "true"` to enable WGE to create the kustomization in cluster creation
+                - It should have the following values that’s needed for **kustomization**
+
+                - GitOps template with sops enabled (check how to add the annotations below)
```

Comment:
I am a bit confused, in the point before this you say "it should have the following values" but you jump straight to this next bullet point.